ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -375,7 +390,7 @@ std::vector<GlobalVector*> setInitialConditions(
 {
     std::vector<GlobalVector*> process_solutions;
 
-    unsigned pcs_idx = 0;
+    int pcs_idx = 0;
```

Comment:
Not from this PR, but there is a difference btw. an index and an id. Which one is used here?

Reply:
Changed to `process_id`.